### ISSUE TITLE
[Bug]: micro select text

### ISSUE THREAD

**Issue by lqvsyi2** [termux/termux-app#4331]
### Problem description

Text selection bug in the text editor based on Micro.

### Steps to reproduce the behavior.

https://github.com/user-attachments/assets/99150e3d-f308-4848-9e9d-24e9de18a76b

### System information

* Termux application version:0.119.0
* Android OS version:Android 13

**Comment by SAKIB-SALIM** [comment]
It's not a bug. You can select text with left click and scrolling with your mouse.

**Comment by niten94** [comment]
I do not think this is a bug and it also happens with other programs used on the terminal emulator of Termux app. I think a plugin in micro where start and end of selection is pressed on screen can be made. I may check, then write a comment again or create thread at https://github.com/zyedidia/micro/discussions when I have time.

There are related issues like #35 and #1384, so I think this can be closed as a duplicate issue. I did not read the comments but I think programs do not receive mouse hold events, because there is currently no method to distinguish it with scroll events.

**Comment by niten94** [comment]
> I think a plugin in micro where start and end of selection is pressed on screen can be made as workaround. I may write a comment again or create thread at https://github.com/zyedidia/micro/discussions sometime.

This is possible but thinking about usage method and implementation is harder than I thought, so it's unlikely that I can make it within months.

I have also thought about adding actions in the plugin where splits can be resized using keyboard, but only the implementation of this is hard.
